<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="Averages" stdset="0">
   <string/>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>82</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Averages</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NSA_SpinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>83</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Angle</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="FA_SpinBox">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>10</y>
     <width>53</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>10</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>deg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>ADC Samples</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ADC_samples_SpinBox">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>42</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>8192</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>42</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>74</y>
     <width>94</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ADC Duration</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ADC_duration_SpinBox">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>74</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>74</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Î¼s</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
